Connected_RgbLed/RgbLedHost/src/RgbLedHost/RgbLedHost/App.cs

```diff
@@ -39,11 +39,11 @@ protected void InitializeWebServer()
         {
             var handler = new RequestHandler();
 
-            handler.TurnOn += _rgbController.TurnOn;
-            handler.TurnOff += _rgbController.TurnOff;
-            handler.StartBlink += _rgbController.Blink;
-            handler.StartPulse += _rgbController.Pulse;
-            handler.StartRunningColors += _rgbController.RunningColors;
+            handler.TurnOn += (objects, args) => { _rgbController.TurnOn(); };
+            handler.TurnOff += (objects, args) => { _rgbController.TurnOff(); };
```

Comment:
instead of `(objects, args)` do `(s, e)`.

it's actually `(sender, eventArgs)`, but `(s, e)` is fine.